README.md

```diff
@@ -1,57 +1,190 @@
+
+```Python
+ALLOWED_HOSTS = ['https://example-domain.azurewebsites.net']
+```
+
+You may **skip the following optional section** if you are running a non-production deployment or you have skipped the optional  in [Step 1: Prepare the web app for deployment](#step-1-prepare-the-web-app-for-deployment)
+<details>
+
+<summary> expand this section if you elected to deploy app secrets more securely.</summary>
+
+You have several options for adding your app secrets more securely. The following two examples demonstrate using Azure Vault **or** environment variables to add secrets removed from the `aad.config.json` file. Other secrets from your settings files should also be added in a similar method.
+
+ 1. **Azure Vault**. Use the [Azure Key Vault Secret client library for Python](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-secrets). Set the client secret value in vault, naming it `CLIENT_SECRET` for example. Then set up the Azure key vault client in your app, and modify the `Sample/settings.py` file as follows:
+
+    ```Python
+    # find the following line in the settings file.
+    AAD_CONFIG = AADConfig.parse_json(file_path='aad.config.json')
+    # directly after the above line, add the following line:
+    AAD_CONFIG.client.client_credential=secret_client.get_secret("secret-name")
+    # Note: secret_client is your fully set up Azure Key Vault Secret client library for Python (https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-secrets). You must set it up as per instructions on the Azure Key Vault Secret client library for Python page.
+    ```
+
+ 2. **Environment Variables** (*Azure Portal > App Services > `Your App` > Configuration*). You must set the value for `CLIENT_SECRET`. Modify the `Sample/settings.py` file as follows:
+
+    ```Python
+    # find the following line in the settings file.
+    AAD_CONFIG = AADConfig.parse_json(file_path='aad.config.json')
+    # directly after the above line, add the following line:
+    AAD_CONFIG.client.client_credential=os.environ.get("secret-name")
+    ```
+
+</details>
+
+Go to the Azure tab on VSCode. Under the `APP SERVICE` toolbox, right click the app name that you created in [Step 2: Prepare the app service and database](#step-2-prepare-the-app-service-and-database) (You may need to refresh the list with the refresh button on the toolbox). Choose `Deploy to Webapp`
+
+## We'd love your feedback!
+
+Were we successful in addressing your learning objective? Consider taking a moment to [share your experience with us](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR73pcsbpbxNJuZCMKN0lURpUM0dYSFlIMzdHT0o3NlRNVFpJSzcwRVMxRyQlQCN0PWcu).
+
+## More information
+
+- [Azure App Services](https://docs.microsoft.com/azure/app-service/)
```

Comment:
few more links would be nice. like azure postgres server, key vault, managed identity etc.